models/article-model.js

```diff
@@ -19,16 +19,30 @@ exports.fetchArticleById = (article_id) => {
   });
 };
 
-exports.allArticlesWithCommentCount = () => {
+exports.allArticlesWithCommentCount = (
+  topic,
+  sort_by = "created_at",
+  order = "DESC"
+) => {
+  const queryValues = [];
+
```

Comment:
This is currently open to SQL injection. We need to think about having some sort of code that ensures we can only pass a valid sort_by or order query.